manifests/install.pp

```diff
@@ -2,9 +2,16 @@
 # @api private
 class katello_devel::install {
   if $facts['os']['release']['major'] == '9' {
-    yumrepo { 'crb':
-      enabled => true,
-      before  => Package['libvirt-devel'],
+    if $facts['os']['name'] == 'RedHat' {
+      rh_repo { "codeready-builder-for-rhel-9-${facts['os']['architecture']}-rpms":
```

Comment:
I saw that in CentOS Stream 10 also has a CRB repo, so I think we can assume RHEL 10 has the same structure (even if line 4 prevents it from being executed).  Should we prepare for that?
```suggestion
      rh_repo { "codeready-builder-for-rhel-${facts['os']['release']['major']}-${facts['os']['architecture']}-rpms":
```